src/App.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import "./App.css";
+import { getSubCategory } from './helpers/getSubCategory';
+import {HOMEPAGE_CONSTANTS}  from "./constants.general";
+import { Route, Routes } from "react-router";
+import Homepage from './pages/homepage/Homepage';
+import Checkout from './pages/checkout/Checkout';
+
+
+class App extends React.Component {
+  
+  constructor(props) {
+    super(props);
+    this.state = {
+      category: HOMEPAGE_CONSTANTS.DEFAULT_CATEGORY,
```

Comment:
We should not hardcode this default value instead we can take first value from backend response